### PR TITLE
Port encrypted cookies to cryptography (fixes #72)

### LINKED ISSUE #72: port cookie sessions to cryptography for Travis speedup
I'm making this placeholder to assign myself https://github.com/brightinteractive/django-encrypted-cookie-session/issues/16 . We need this because Travis running with sudo is an abominable snowman of slowness. Not having to compile M2Crypto will simplify a lot of other things too :)

### PR DESCRIPTION
**Pull request by kumar303**
Depends on https://github.com/brightinteractive/django-encrypted-cookie-session/pull/18

Fixes #72

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 FROM mozillamarketplace/centos-mysql-mkt:0.2
```
Comment (andymckay) [inline Dockerfile]: we should use :latest here instead

File: Dockerfile [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
 RUN yum install -y supervisor bash-completion && yum clean all
+# This is for the cryptography module:
+RUN yum install -y gcc libffi-devel python-devel openssl-devel && yum clean all
```
Comment (andymckay) [inline Dockerfile]: I think you'll find gcc is already installed in a previous container, I think python-devel is also there. Not sure about the others though.

https://github.com/mozilla/marketplace-env/blob/master/docker/base-images/python27/Dockerfile#L5

File: requirements/compiled.txt [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-M2Crypto==0.22.3
-
 # sha256: bOShGrSFpnqJmqPggMl7IqnfQMcHe9kUaoIdOHLPIDc
 lxml==3.4.3
+
+# sha256: OQlwtgJwjJHdxzlTu2kp5WKRwYpNgPNgr6APrYtvMzk
+cffi==1.1.2
+
+# sha256: vJ3dFELOZ6W71VmOztEn3gANpV8M4lSYsRdHRsApCzw
+cryptography==0.9.2
```
Comment (andymckay) [inline requirements/compiled.txt]: A wheel of these would be super nice.

Comment (kumar303) [inline requirements/compiled.txt]: oh yeah, I was going to ask about how to do that ;)

File: Dockerfile (hunk 2 above)
Comment (kumar303) [inline Dockerfile]: hmm, I was getting errors installing cryptography but let me double check what's needed.

**Pull request merged**